cluster-management/setup/update-strategies/index.md

```diff
@@ -113,21 +115,53 @@ A timeframe in which to update can be specified by using two systemd units, a ve
 Description=Reboot if an update has been downloaded
 
 [Service]
-ExecStart=/usr/bin/bash -c 'if update_engine_client -status | grep NEED_REBOOT; then reboot; fi'
+ExecStart=/usr/bin/bash /opt/bin/update-window.sh
 ```
 
 #### update-window.timer
 
 ```yaml
 [Unit]
-Description=Reboot if needed at 05:00 daily
+Description=Reboot timer
 
 [Timer]
-OnCalendar=*-*-* 05:00:00
-```
+OnCalendar=*-*-* 05,06:10/20:00
```

Comment:
From #421, try to find a workaround  so systems have a chance to reboot in the same maintenance window instead of delaying to next window. For a cluster of 3 etcd servers, 1 reboot slot, it would take 3 days. For simplicity, yes leave 05:00:00, but maybe mention the possible delay in documentation.  Or maybe "05,06:00/30:00" schedule is easier to understand - 4 runs within two hours probably will cover a lot of installation cases. I am not sure. It depends on size and reboot slot.